specification/2.0/README.md

```diff
@@ -249,6 +249,16 @@ All angles are in radians.
 
 Positive rotation is counterclockwise.
 
+The [node transformations](#transformations) and [animation channel paths](#animations) are 3D vectors or quaternions with the following data types and semantics:
+
+|Property|Type|Component Type(s)|Description|
+|----|----------------|-----------------|-----------|
+|translation|`"VEC3"`|`5126`&nbsp;(FLOAT)|XYZ translation vector|
+|rotation|`"VEC4"`|`5126`&nbsp;(FLOAT)<br>`5120`&nbsp;(BYTE)&nbsp;normalized<br>`5121`&nbsp;(UNSIGNED_BYTE)&nbsp;normalized<br>`5122`&nbsp;(SHORT)&nbsp;normalized<br>`5123`&nbsp;(UNSIGNED_SHORT)&nbsp;normalized|XYZW rotation quaternion|
```

Comment:
I think only FLOAT is allowed?

From https://github.com/javagl/glTF/blob/e637a434badd7558dd70bc54ed0e863f58d59a5d/specification/2.0/README.md#transformations
> rotation is a FLOAT_VEC4 unit quaternion value, (x, y, z, w), in the local coordinate system.

Reply:
Oh, I see where you got that from. Looks like we have a discrepancy here. TRS on a node are stored in the JSON where as TRS stored for animations are in a buffer.

Reply:
Indeed, the component types refer to the animation channel paths. Should I just omit this column, making it just  

    Property     Type?        Description
    translation  3D vector    Translation (xyz)...
    rotation     4D vector    Quaternion (xyzw)...
    scale        3D vector    Scaling (xyz)...
    
or maybe omit the `Type` altogether? (Then I'd likely convert it into a bulletpoint list...)

Reply:
I don't have a strong preference. Bulleted list will be okay with me without the types. People who want the specifics should look at the corresponding links.

@lexaknyazev On a side note (I'm looking at the supported list for animation quaternions and weights), is normalized UNSIGNED_BYTE/UNSIGNED_SHORT useful for quaternions and normalized BYTE/SHORT useful for weights? Only positive values for quaternion seem unlikely and I believe weights are always positive?

Reply:
Afair, we've decided to not limit those since exporters could optimize data for some cases while implementations will likely be able to read all kinds of accessors anyway.